Five lab

yuriivynar opened the pull request.

Vectaro commented:
![TheboysPerfectGIF](https://github.com/yuriivynar/lab5-programming/assets/83184499/241534ee-32bc-4340-8fb2-c86ef083319c)

zveres-ss reviewed (changes requested):
that allows me to convert time in the format hh:mm:ss to the number of seconds and vice versa

can't find second method in the code - lab is not complete

yuriivynar commented:
> that allows me to convert time in the format hh:mm:ss to the number of seconds and vice versa
> 
> can't find second method in the code - lab is not complete

Thanks for the comment

yuriivynar commented:
> that allows me to convert time in the format hh:mm:ss to the number of seconds and vice versa
> 
> can't find second method in the code - lab is not complete

Thanks for the comment! 
I implemented the two conditions of this `laboratory` using one method with "If & else". Since under PEP8 you can't put "else" after "returns", my code for the second condition (converts from seconds to format hh:mm:ss ) looks like this. In the condition of the laboratory, it was not said exactly how to implement this class, so I implemented it with one method and consider the laboratory completed. I hope I understood you correctly and answered your comment. 
![Знімок екрана (83)](https://github.com/yuriivynar/lab5-programming/assets/151685418/ea89f082-5acd-40cf-8d51-d45764fe2353)
